app/controllers/renewal/InvolvedInOtherController.scala

```diff
@@ -54,25 +54,19 @@ class InvolvedInOtherController @Inject()(
         }
   }
 
-  private def businessTypes(activities: BusinessMatching): Option[String] = {
-    val typesString = activities.activities map { a =>
-      a.businessActivities.map {
-        case AccountancyServices => Messages("businessmatching.registerservices.servicename.lbl.01")
-        case BillPaymentServices => Messages("businessmatching.registerservices.servicename.lbl.02")
-        case EstateAgentBusinessService => Messages("businessmatching.registerservices.servicename.lbl.03")
-        case HighValueDealing => Messages("businessmatching.registerservices.servicename.lbl.04")
-        case MoneyServiceBusiness => Messages("businessmatching.registerservices.servicename.lbl.05")
-        case TrustAndCompanyServices => Messages("businessmatching.registerservices.servicename.lbl.06")
-        case TelephonePaymentService => Messages("businessmatching.registerservices.servicename.lbl.07")
+  private def businessTypes(activities: BusinessMatching): Option[List[String]] = {
+    val typesString: Option[List[String]] = activities.createBusinessTypeOrderedList
+
+    typesString.map {
+      case t =>
+        t.map(item => {
+        if (item.toLowerCase.startsWith("a") || item.toLowerCase.startsWith("e")) {
+          s"an ${item.toLowerCase}"
+        } else {
+          s"a ${item.toLowerCase}"
+        }
+        })
```

Comment:
In my personal opinion I think this method is a lot harder to read/understand than the previous method. We'd also have to come back to change this solution if we ever had to add a new section that began with another vowel other than E and A. It boils down to personal preference I guess but I personally prefer how it's written on the left. All that would need to be changed is the order of case statements which is what we have done in previous content tickets.

Reply:
I would keep createBusinessTypeOrderedList within this file.
Does it need to be a `def`, or can we create it as a `val`? I would much prefer it to be a `val` if possible.

Reply:
After discussion, I agree that createBusinessTypeOrderedList is in the right place.
I would rename it to something more readable and evocative of what it is achieving.

Reply:
@matthyatt1 @drewmiley refactoring is done